docs/content/any/project/changelogs/2021-11-30.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Release 2021-11-30
+menuTitle: "2021-11-30"
+any: true
+description: >-
+  Changelog for Release 2021-11-30 (0.24.0) containing new features,
+  bug fixes, and more.
+---
+
+## Oso 0.24.0
+
+### VSCode Extension
+
+- Our VSCode Extension now shows displays inline Diagnostics (errors & warnings) in the editor as well as in VSCode's **Problems** pane.
+  - The extension immediately highlights errors and warnings encountered while parsing and validating your policy, such as if a rule is missing a trailing semi-colon, a resource block declares `"owner"` as both a role and a relation, or your policy contains no `allow()` rule. You would normally see this feedback when running your application, but the extension surfaces it while you edit your policy.
+  - The extension collects **non-identifiable** telemetry data that we use to improve Oso. We collect data into un-timestamped batches instead of sending it on every policy load since we care about aggregate statistics; not tracking your personal development behavior. **We will never sell this data**. For more info on exactly what is tracked and why, see [this page][https://docs.osohq.com/reference/tooling/ide/telemetry.html] in our docs.
```

Comment:
The `[this page]` link should be a relative link since this is within our Hugo ecosystem. If it's a relative link, it'll link to `localhost` if you're developing locally or to `docs.osohq.com` once it's deployed